### DISCUSSION TITLE
Invalidate/Revoke a license

### DISCUSSION THREAD

**Discussion by craigreilly** [Willy-Kimura/TrialMaker.Demo#5] [Need Help]
As a test, I licensed a program on a laptop. I later deleted the license from "License Generator".
But the app continues to work. IS there a way to revoke a license?

How about the ability to move the license to another machine?

**Comment by Willy-Kimura** [comment]
Hi @craigreilly 

The standard product only supports offline activation, which means that deleting a license from the License Generator will simply remove the record from the local database and not from the remote client's machine. This feature requires the [online activation](https://willykimura.gumroad.com/l/qmWSh) product package.

On your second question, yes you can move the license to another machine by regenerating a license based on the other machine's Hardware ID (due to its node-locked licensing mechanism) and the license will be transferred once activated in the machine. An easier option would be to include the other machine's Hardware ID during license generation (since the product supports multi-device activation) then apply the code to the other machine from the application's activation window.
